dreambooth_google_colab_joepenna.ipynb

```diff
@@ -219,7 +230,8 @@
     "from google.colab import drive\n",
     "drive.mount('/content/drive')\n",
     "\n",
-    "!cp trained_models/{file_name.ckpt} /content/drive/MyDrive/{file_name.ckpt}"
+    "# copy all ckpt files to google drive root dir\n",
+    "!cp trained_models/*.ckpt /content/drive/MyDrive"
```

Comment:
Nice fix, this colab predates the save_every_x_steps stuff so this is a great find.